lib/rules/no-unsupported-features/node-builtins.js

```diff
@@ -31,6 +31,9 @@ const trackMap = {
             doesNotReject: { [READ]: { supported: "10.0.0" } },
             notDeepStrictEqual: { [READ]: { supported: "4.0.0" } },
             rejects: { [READ]: { supported: "10.0.0" } },
+            CallTracker: {
+                [READ]: { supported: null, experimental: "14.2.0" },
+            },
```

Comment:
off topic: is there a way to fetch the supported version from node.js repo, rather than maintaining a hard-code versions?

Reply:
I looked for such a method, but could not find it... But I created https://github.com/sosukesuzuki/nodejs-api-table instead. This is a table generated by fetching and parsing the Markdown of Node.js documentation, and can be viewed in a browser. It's a little better than before.
But if there's an even better way, I'd like to know.